### PR TITLE
multi-cluster updates

### PR DESCRIPTION
**Pull request by jessegoodier**
## Proposed Changes
- clean up multi-cluster.md
- add doc and diagram for kubecost diagnostics

todo: 
- add to nav

@bstuder99 What will the URL be?
https://docs.kubecost.com/install-and-configure/install/diagnostics

cc: @thomasvn

### CODE REVIEW

**Comment by thomasvn** [comment]
@jessegoodier Made some edits. Most significantly, I changed the example Helm config we show in the docs. And I updated the API docs to be in Swagger format. Let me know if you think anything needs modification or reverting.

I'm also going to mark this PR as ready for review.

**Review by bstuder99 — APPROVED** [review]

File: install-and-configure/install/multi-cluster/diagnostics.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Diagnostics API
+
+The diagnostics API can be accessed through `/model/multi-cluster-diagnostics`
+
+The `window` query parameter is required, which will return all diagnostics within the specified time window.
+
+Example output:
+
+```json
```
[the remark this answers is not in the document]
Comment (jessegoodier) [reply install-and-configure/install/multi-cluster/diagnostics.md:63]: Make sure you are looking at the complete PR commit.

[the remark this answers is not in the document]
Comment (jessegoodier) [reply install-and-configure/install/multi-cluster/diagnostics.md:63]: my bad, this is last month. resolved.

File: install-and-configure/install/multi-cluster/diagnostics.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Diagnostics architecture
+
+{% hint style="info" %}
+In the below diagram, the arrows originate from the the pod initiating the request and point to the resource that receives the request.
+This diagram is specific to the requests required for diagnostics only. For additional diagrams, see the [multi-cluster guide](multi-cluster.md).
+{% endhint %}
+
+![Kubecost-Agent-Diagnostics](/images/daigrams/Agent-Diagnostics-Architecture.png)
+
+## Diagnostics API
```
[the remark this answers is not in the document]
Comment (jessegoodier) [reply install-and-configure/install/multi-cluster/diagnostics.md]: Added in current doc.

**Pull request merged**